libs/remix-ui/workspace/src/lib/components/file-explorer.tsx

```diff
@@ -97,6 +101,100 @@ export const FileExplorer = (props: FileExplorerProps) => {
     }
   }, [treeRef.current])
 
+  useEffect(() => {
+    const performDeletion = async () => {
+      const path: string[] = []
+      if (feTarget?.length > 0 && feTarget[0]?.key.length > 0) {
+        feTarget.forEach((one) => {
+          path.push(one.key)
+        })
+        await deletePath(path)
+      }
+    }
+
+    if (treeRef.current) {
+      const deleteKeyPressHandler = async (eve: KeyboardEvent) => {
+        if (eve.key === 'Delete' ) {
+          (window as any)._paq.push(['trackEvent', 'fileExplorer', 'deleteKey'])
```

Comment:
it will be better to create a variable for `(window as any)`

Reply:
also `_paq.push` takes 4 args

Reply:
That's right.